connectors/.env.example

```diff
@@ -1,4 +1,6 @@
 CONNECTORS_DATABASE_URI=CONNECTORS_DATABASE_URI=postgres://postgres:mysecretpassword@localhost:5432/postgres
 NANGO_SLACK_CONNECTOR_ID=nango_slack_integration_id
 NANGO_SECRET_KEY=sk...
-DUST_CONNECTORS_SECRET=L3AhrnR3YuYEx6otwUnbzit29C1hTYnp
\ No newline at end of file
+DUST_CONNECTORS_SECRET=L3AhrnR3YuYEx6otwUnbzit29C1hTYnp
+DUST_CONNECTORS_WEBHOOKS_SECRET=webhooks_secret
```

Comment:
How is `DUST_CONNECTORS_WEBHOOKS_SECRET` set up with Slack? How is the webhook URL set up in Slack?

Please add a comment that explains how everything works or in the initial design doc or a runbook, really up to you.

Reply:
Started the beginning of a slack runbook:
https://www.notion.so/dust-tt/d151a5226dba4f8fbe094365da5f00d4?v=c3d749040f7143f7aa61b0d29ba53a83&p=e59f50c458a64a9ea64f4fd17dacaffb&pm=s

Reply:
Great!